monitor/grafana/zeebe.json

```diff
@@ -503,13 +503,13 @@
                 "uid": "${DS_PROMETHEUS}"
               },
               "editorMode": "code",
-              "expr": "max(zeebe_blacklisted_instances_total{namespace=~\"$namespace\", pod=~\"$pod\", partition=~\"$partition\"}) by (namespace)",
+              "expr": "max(zeebe_blacklisted_instances_total{namespace=~\"$namespace\", pod=~\"$pod\", partition=~\"$partition\"}) by (namespace) \nor max(zeebe_banned_instances_total{namespace=~\"$namespace\", pod=~\"$pod\", partition=~\"$partition\"}) by (namespace)",
```

Comment:
:bulb:  TIL about `or`. If I understand correctly, it matches two "vectors" based on the labels. For us that means on the namespace label. If a cluster is updated to a version which uses the new naming, the old time series does not receive new data and we will use the new time series instead. Makes sense :+1: